### PR TITLE
Apache NetBeans 20 release

### PR DESCRIPTION
**Pull request by ebarboni**
put v19 in archive
put v20 live
archive page updated

### CODE REVIEW

File: modules/ROOT/pages/download/nb20/index.adoc [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+include a local JDK runtime for the IDE to run on, for a self-contained out-of-the-box
+experience.
+
+IMPORTANT: Individual NetBeans committers may provide additional binary packages as a convenience.
+While built using the Apache NetBeans release, they are not releases of the Apache Software
+Foundation. They may include other contents (eg. JDK) under additional license terms.
+
+== Deployment Platforms
+
+The Apache NetBeans {netbeans-version} binary releases require JDK 11+, and officially support running on JDK 11, 17 and 20.
```
Comment (mbien) [inline modules/ROOT/pages/download/nb20/index.adoc]: change 20 to 21 please

Comment (ebarboni) [reply modules/ROOT/pages/download/nb20/index.adoc]: fixed

**Review by mbien — APPROVED** [review]
quickly checked the links - looks good!

**Comment by ebarboni** [comment]
added the blog entry

File: modules/ROOT/pages/blogs/entry/announce-apache-netbeans-20-released.adoc [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+:page-syntax: true
+
+
+The Apache NetBeans team is pleased to announce that Apache NetBeans 20 was released on December 1, 2023.
+
+What's in the Apache NetBeans 20 release:
+
+https://github.com/apache/netbeans/releases/tag/20
+
+With thanks to 26 contributors, including 6 who have contributed for the first time. Thank you!
```
Comment (mbien) [inline modules/ROOT/pages/blogs/entry/announce-apache-netbeans-20-released.adoc:39]: esp thanks to dependabot who made his first contribution

**Review by mbien — APPROVED** [review]

File: modules/ROOT/pages/blogs/entry/announce-apache-netbeans-20-released.adoc (hunk 2 above)
Comment (ebarboni) [reply modules/ROOT/pages/blogs/entry/announce-apache-netbeans-20-released.adoc:39]: was not already there ?

Comment (mbien) [reply modules/ROOT/pages/blogs/entry/announce-apache-netbeans-20-released.adoc:39]: no its the first time that we merged a dependabot PR :P

**Pull request merged**